setup.py

```diff
@@ -59,7 +30,6 @@ def run_tests(self):
 
 # bdist_wheel
 extras_require = {
-    'test': tests_require,
```

Comment:
The Fedora package uses the `[test]` extra to fetch test dependencies. I'd appreciate if this could stay.

Reply:
Sure, I'll revert that change.